types.go

```diff
@@ -27,8 +31,28 @@ type ForwardMsg struct {
 	Payload interface{} `json:"payload"`
 }
 
+type SServerResponse struct {
+	Response   interface{}
+	StatusCode int
+}
+
 type SignalingServer interface {
-	NewConnection(msg NewConnMsg) NewConnReply
-	Forward(id string, msg ForwardMsg) error
-	Messages(id string, start int, count int) ([]interface{}, error)
+	// All endpoints in the SignalingServer return the (possibly modified)
+	// response from the Host Orchestrator and the status code if it was
+	// able to communicate with it, otherwise it returns an error.
+	NewConnection(msg NewConnMsg) (SServerResponse, error)
+	Forward(id string, msg ForwardMsg) (SServerResponse, error)
+	Messages(id string, start int, count int) (SServerResponse, error)
+}
+
+type DeviceDesc struct {
+	// The (internal) address of the host where the cuttlefish device is running
+	Addr string
```

Comment:
let's add an example to have an idea about the format these address values will have.

Reply:
Modified the comment to be more explicit about it returning a network address instead of a URL and clarified that either domain names and IP addresses (v4 and v6) are OK.